flowapi/flowapi/query_endpoints.py

```diff
@@ -409,3 +413,202 @@ async def get_available_dates():
     else:
         assert reply["status"] == "error"
         return {"status": "error", "msg": reply["msg"]}, 500
+
+
+@blueprint.route("/qa/<cdr_type>/<check_id>", methods=["GET"])
+@jwt_required
+async def get_qa_date_range(cdr_type, check_id):
```

Comment:
Can we move these to a separate file e.g. `qa_endpoints.py`, and then register the blueprint as prefix "/api/0/qa" - they're quite distinct in purpose from the main query ones.